x/bank/src/genesis.rs

```diff
@@ -29,9 +28,6 @@ impl Default for GenesisState {
                 send_enabled: vec![],
                 default_send_enabled: true,
             },
-            //TODO: this denom metadata should not be hard coded into the bank module
-            // this has been added here for short term convenience. There should be a
```

Comment:
Also remove the commented out lines below?

Reply:
Only when you or me updates init scripts to set metadata in genesis